docs/physics/montecarlo/initialization.ipynb

```diff
@@ -71,10 +71,39 @@
   },
```

Comment:
<p>Try this with just numba, and you won't need/want to change anything within the function from how it was before, only the decorator</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tardis-sn/tardis/pull/2270/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>